Testscripts/Linux/utils.sh

```diff
@@ -2226,7 +2226,7 @@ function install_epel () {
 					epel_rpm_url="https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm"
 				elif [[ $DISTRO_VERSION =~ ^7\. ]]; then
 					epel_rpm_url="https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
-				elif [[ $DISTRO_VERSION =~ ^8\. ]]; then
+				elif [[ $DISTRO_VERSION == "8.0" ]]; then
```

Comment:
@LiliDeng This might be different issue here, but want to ask. I have been chasing epel repo access issue in redhat 8.1 because this blocks stress-ng and other required package installation in LISAv2.  Do you have any information redhat epel is only available for 8.0 not the later version? What is the background?